assets/app/views/_alerts.html

```diff
@@ -1,20 +1,32 @@
 <div ng-repeat="alert in alerts">
   <div ng-switch="alert.type">
-    <div ng-switch-when="error" class="alert alert-danger">
+    <div ng-switch-when="error" class="alert alert-danger" ng-class="{'alert-dismissable': alert.dismissable}">
+      <button ng-if="alert.dismissable" type="button" class="close" data-dismiss="alert" aria-hidden="true">
```

Comment:
I believe the `ng-repeat` should treat each alert separately, the `ng-switch` should cycle for individual alerts.  So at this point you should get stacked alerts of the same type, not a queue (which is prob good, we don't have a priority system).